src/pages/uxp/reference-js/Global Members/HTML Elements/HTMLWebViewElement.md

```diff
@@ -0,0 +1,711 @@
+
+<a name="htmlwebviewelement" id="htmlwebviewelement"></a>
+
+# window.HTMLWebViewElement
+**Properties**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| uxpallowinspector | `boolean` | Enable Developer tools for WebView <br>&nbsp;&nbsp;e.g., \<webview src="https://www.adobe.com" uxpallowinspector="true"\>\<\/webview\> |
+
+
+
+<a name="htmlwebviewelement-src" id="htmlwebviewelement-src"></a>
+
+## src : `string`
+The url of the webview
+
+
+
+<a name="htmlelement-dataset" id="htmlelement-dataset"></a>
+
+## dataset
```

Comment:
Missing type, according to https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset should be `DOMStringMap` (assuming that we're compatible with these specs)

Reply:
🟡

Reply:
Tracked in #76